src/Components/Dashboard/MiniDashboard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { WithToken } from "../../HOCs/withToken";
+import { useEffect } from "react";
+import styled from "styled-components";
+import { useGetUser } from "../apis/useGetUser";
+import { set } from "lodash";
+import { useState } from "react";
+
+const MiniDashboardComponent = () => {
+  const [res,getUser]=useGetUser()
+  const [user,setUser]=useState()
```

Comment:
why are we setting user at component state here when we have a user context?

Reply:
resolved